src/IO/Observer/ReductionActions.hpp

```diff
@@ -31,6 +31,10 @@
 #include "Utilities/TaggedTuple.hpp"
 
 namespace observers {
+
```

Comment:
Please enclose this in:
```cpp
/// \cond
template <class Metavariables>
struct ObserverWriter;
/// \endcond
```

The reason is that Doxygen otherwise finds the undocumented versions (sometimes) and adds them to the generated documentation cluttering it up.